### DISCUSSION TITLE
Security Onion 2.4 - SOC Customization

### DISCUSSION THREAD

**Discussion by YvesEutelsat** [Security-Onion-Solutions/securityonion#13072] [2.4] [answered]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

airgap

### Hardware Specs

Meets minimum requirements

### CPU

16

### RAM

128000

### Storage for /

1Tb

### Storage for /nsm

300Mb

### Network Traffic Collection

tap

### Network Traffic Speeds

more than 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

Good day,

I'm trying to customization my SOC overview & login banner page.  Going on Web Interface under, soc - files - soc, I was able to modify the text using Markdown guide.  I would like to add a picture of my organization.  Doing some research, I wrote this ![Picture description](/tmp/picturename.jpg).  Only thing that appears, it is a broken image icon.  Just wondering if Security Onion can display picture, if so how should I do it?  Or just I copied the picture into another location?

I have tried different image extension (.jpg .bmp .png .gif)

Much appreciated
Thanks for your assistance

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment, marked as answer]
You can add images but they must be hosted from another host that is accessible by the user’s browser. For example, let’s use one of the images from our online documentation. The markdown to add that image would look like this:
```
![SOC Dashboards](https://docs.securityonion.net/en/2.4/_images/51_dashboards.png)
```
